src/js/components/pollListItem.vue

```diff
@@ -299,4 +333,58 @@ export default {
 	background-color: var(--color-success);
 	border-radius: 50%;
 }
+
+
+.thumbnail.accessasd {
```

Comment:
is the "asd" at the end on purpose? :grin:

Reply:
Hehe. Was left from playing with the css attributes.